### PR TITLE
Update "content-based message routing" and "transforming message formats" tutorials

### PR DESCRIPTION
**Pull request by dulajdilshan**
## Purpose
$title

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: transforming-message-formats/main.bal [hunk 1]
```diff
@@ -80,16 +81,19 @@ service /healthcare on new http:Listener(port) {
...
-        phone: healthcareReservation.phone,
-        email: healthcareReservation.email
-    },
-    doctor: healthcareReservation.doctor,
-    hospital: healthcareReservation.hospital,
-    appointment_date: healthcareReservation.appointment_date
-};
+isolated function transform(HealthcareReservation reservation) returns HospitalReservation =>
+    let var ssn = from int part in reservation.ssn
+        select part
```
Comment (MaryamZi) [inline transforming-message-formats/main.bal]: Why do we need the query expression?
```suggestion
    let var ssn = reservation.ssn
```

File: transforming-message-formats/main.bal [hunk 2]
```diff
@@ -8,35 +8,36 @@ final http:Client hospitalServiceEP = check initializeHttpClient();
...
 function initializeHttpClient() returns http:Client|error => new (hospitalServiceUrl);
 
-type HealthcareReservation record {|
-    string name;
+type HealthcareReservation record {
+    string firstName;
+    string lastName;
     string dob;
-    string ssn;
+    int[] ssn;
```
Comment (MaryamZi) [inline transforming-message-formats/main.bal]:
```suggestion
    int[3] ssn;
```

File: transforming-message-formats/main.bal [hunk 3]
```diff
@@ -8,35 +8,36 @@ final http:Client hospitalServiceEP = check initializeHttpClient();
...
     string dob;
     string ssn;
     string address;
     string phone;
     string email;
-|};
+};
 
-type HospitalReservation record {|
+type HospitalReservation record {
```
Comment (MaryamZi) [inline transforming-message-formats/main.bal]: Can we make these closed?

Comment (dulajdilshan) [reply transforming-message-formats/main.bal]: These are generated records. However, We can make this change to the source code

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**